library/Backends/llvm/src/Juvix/Backends/LLVM/Parameterization.hs

```diff
@@ -9,11 +9,24 @@ module Juvix.Backends.LLVM.Parameterization
   )
 where
 
+import qualified Juvix.Backends.LLVM.Codegen.Types as Types
 import Juvix.Backends.LLVM.Primitive
 import qualified Juvix.Core.Base.Types as Core
 import qualified Juvix.Core.IR.Evaluator as IR
 import qualified Juvix.Core.Parameterisation as Param
 import Juvix.Library
+  ( Applicative (pure),
```

Comment:
Is this intended?

Reply:
It was the pre-commit script that introduced it, so I think that if we didn't, then the script would just try to re-introduce into future PRs.  Maybe I'm misunderstanding something, though.